pkg/kv/kvserver/replica_closedts_test.go

```diff
@@ -722,3 +727,125 @@ func BenchmarkBumpSideTransportClosed(b *testing.B) {
 		}
 	}
 }
+
+// TestNonBlockingReadsAtResolvedTimestamp tests that reads served at or below a
+// key span's resolved timestamp never block or redirect to the leaseholder. The
```

Comment:
In this test are we testing that these requests aren't being redirected to the leaseholder?